Packs/PrismaCloudCompute/Playbooks/Prisma_Cloud_Compute_-_Get_Defender_Logs.yml

```diff
@@ -0,0 +1,341 @@
+id: Prisma Cloud Compute - Get Defender Logs
```

Comment:
I think we should work with playbook outputs and not with set and map actions inside the playbook. It will make the use of the playbook easier to the next people who will use it.